src/feed-list.scss

```diff
@@ -0,0 +1,127 @@
+@import "./mixins";
+
+$block: #{$fd-namespace}-feed-list;
+
+.#{$block} {
+  $fd-feed-list-padding-start: 1rem;
+  $fd-feed-list-padding-end: 0.5rem;
+  $fd-feed-list-margin-start: 0.5rem;
+  $fd-feed-list-margin-end: 0;
+  $fd-feed-list-icon-color: var(--sapContent_IconColor);
+
+  @include fd-reset();
+  @include fd-flex();
+
+  background-color: var(--sapField_Background);
+  max-width: 40rem;
+
+  .#{$block}__thumb {

```

Comment:
Thumbnail should have `0.75rem`, not `0.5rem` margin between text
![image](https://user-images.githubusercontent.com/26483208/95840625-ab6e4580-0d44-11eb-880d-c7990c849078.png)